hatch-studio/react-datasheet
# Shift+arrow selection fixes

To test:
1. Select a cell
2. Hold the `shift` key to select neighboring cells
3. It should work as expected (Excel/Google Sheets style), including stopping at the edge of the table

Notes:
* Added a highlight for the `start` cell, hooked to a CSS class of `is-start`
